locale/en/LC_MESSAGES/django.po

```diff
@@ -381,8 +381,8 @@ msgstr "external"
 #: course/models.py
 msgid "EXTERNAL_USER_DESCRIPTION"
 msgstr ""
-"The user profile doesn't have a student number, thus the user has logged in "
-"from a different organization or via social authentication"
```

Comment:
I believe in the past the very general (and strange) term of "social authentication" has been used in case we intend to add other forms of social authentication, such as Facebook. Of course, since we don't support others, it could be "trimmed" down to just say Google.
Good point about the student IDs!  Are the users that have logged in via Haka and have a different organization that the local one defined as external?

Reply:
Would it be better to use ["social login"](https://en.wikipedia.org/wiki/Social_login) rather than "social authentication", as it would still be general if we later on decided to add on, say Microsoft login?

Reply:
like you said, external users are now defined from the new organisation field of the UserProfile. Only one organisation counts as internal (set in settings.py). All Haka users have some organisation. Google users have no organisation.

"Social login" is fine for me. At the moment Google is the only method, but indeed there could be more. We aren't planning to add any new social login providers, though.

Reply:
How about combining "from a different organization" and "social authentication/login" into "external authentication" or "external login" as suggested below?

Reply:
good idea